### PR TITLE
add a new version containing srcs in cc_library for the tinyobjloader

### PR DESCRIPTION
**Pull request by shabanzd**
This version has to changes compared to the `2.0.0-rc1` version:
* removes the warning flag that is problematic on some clang versions.
* add srcs to the `cc_library`.

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @Vertexwahn, modules you maintain (tinyobjloader) have been updated in this PR. Please review the changes.

**Comment by shabanzd** [comment]
needs the `skip-url-stability-check` and a maintainer review to pass CI

File: modules/tinyobjloader/2.0.0-rc1.bcr1/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module(
+    name = "tinyobjloader",
+    version = "2.0.0-rc1.bcr1",
```
Comment (Vertexwahn) [inline modules/tinyobjloader/2.0.0-rc1.bcr1/MODULE.bazel:3]: should be  `version = "2.0.0-rc1.bcr.1",` (rename folder accordingly and all other occurances)

File: modules/tinyobjloader/metadata.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
         "github:tinyobjloader/tinyobjloader"
     ],
     "versions": [
-        "2.0.0-rc1"
+        "2.0.0-rc1",
+        "2.0.0-rc1.bcr1"
```
Comment (Vertexwahn) [inline modules/tinyobjloader/metadata.json:15]: `2.0.0-rc1.bcr.1`

File: modules/tinyobjloader/2.0.0-rc1.bcr1/patches/add_module_dot_bazel.patch [hunk 3]
```diff
@@ -0,0 +1,13 @@
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -0,0 +1,10 @@
++module(
++    name = "tinyobjloader",
++    version = "2.0.0-rc1.bcr1",
```
Comment (Vertexwahn) [inline modules/tinyobjloader/2.0.0-rc1.bcr1/patches/add_module_dot_bazel.patch:6]: 2.0.0-rc1.bcr.1

**Review by Vertexwahn — COMMENTED** [review]
Version should be `2.0.0-rc1.bcr.1`

**Comment by bazel-io** [comment]
Hello @sergeykhliustin, modules you maintain (bazelpods) have been updated in this PR. Please review the changes.

**Comment by ziad-fernride** [comment]
@Vertexwahn : sorry was offline the past days ... updated the version string according to the comment

**Review by Vertexwahn — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**